dependencies/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-if(NOT USE_SYSTEM_DATE)
+if(NOT TARGET date::date)
```

Comment:
After thinking about it we might should make that an or condition with what we had before? 

It works this way currently, because the find_package call is before the `add_subdirectory(dependencies)` call. But that doesn't seem very robust.